translator/reader/read_geometry.cpp

```diff
@@ -1044,7 +1044,7 @@ void UsdArnoldReadVolume::Read(const UsdPrim &prim, UsdArnoldReaderContext &cont
         VtValue vdbFilePathValue;
 
         if (vdbAsset.GetFilePathAttr().Get(&vdbFilePathValue, time.frame)) {
-            std::string fieldFilename = VtValueGetString(vdbFilePathValue, &prim);
+            std::string fieldFilename = VtValueGetString(vdbFilePathValue, nullptr);
```

Comment:
shouldn't this have the attribute `vdbAsset.GetFilePathAttr()` instead of nullptr ?

Reply:
It should ! I'll push an update.